README.md

```diff
@@ -1,13 +1,25 @@
 # My Little Rainbow
 
-In this lab, we're going to make a rainbow with HTML `<div>` elements. And while we do it we're going to learn about HTML elements, CSS styling, CSS selectors, how color works in CSS, and importing stylesheets.  That might sound like a lot but it isn't.
+## Problem Statement
 
-Before we start, here's some basic info about HTML and CSS. Skip to [Making a Rainbow](#making-a-rainbow) if you feel comfortable enough with HTML and CSS.
+In this lab, we're going to make a rainbow with HTML `<div>` elements. And while
+we do it we're going to learn about HTML elements, CSS styling, CSS selectors,
+how color works in CSS, and importing stylesheets.
 
-## HTML Basics
-Hyper Text Markup Language, or HTML, is a way to demarcate a document into different parts. Each part is _marked_ by elements (using tags). Each element has its own special connotation that the browser uses to _render_ the HTML document. Use this [cheat sheet](http://www.cril.univ-artois.fr/~lecoutre/teaching/web/sheets/HTML.pdf) on HTML elements for guidance.
+Before we start, here's some basic info about HTML and CSS. Skip to 
+[Making a Rainbow](#making-a-rainbow) if you feel comfortable enough with HTML
+and CSS.
+
+## Review HTML Basics
+
+Hyper Text Markup Language, or HTML, is a way to demarcate
```

Comment:
eek, demarcate? Other docs have the phrase

"is a way to apply _markup_ to _content_"